CMakeLists.txt

```diff
@@ -11,7 +11,7 @@
 #
 ################################################################################
 
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
is this ok to change the pre-requisites? I'm on cmake 3.10 on Ubuntu 18, but we may have dev on version below

Reply:
I Think it's a mistake on my part

Reply:
done